### ISSUE TITLE
JavaScript Course: Update codeblock delimiters per layout style guide

### ISSUE THREAD

**Issue by aconstas** [TheOdinProject/curriculum#26942]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

Codeblocks should be updated to align with new changes in our [layout style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

In the lessons named in this issue, codeblocks will be opened by a set of 3 tildes (the ~ character) and then closed by another set of 3 tildes. To align with the style guide, these tilde characters need to change to backticks (the ` character). Be sure to refer to [the codeblocks section](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#codeblocks) of the style guide for an example of what this should look like.

### Guidance:

 - Comment on this issue to get a lesson assigned to you. You can also ask for a specific one
 - Please wait until you are assigned by a maintainer before starting to work on this issue to avoid multiple people working on the same lessons
 - Make sure to read the [contributing guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md) before starting your first contribution
 - Please add the following to your PR in the Issues section: Related to #xxxxx, since this issue is divided into sections, individual PRs should **NOT INCLUDE** the closes keyword.

### Lessons to update
- [x] @VMadhuranga [Async and Await](https://www.theodinproject.com/lessons/node-path-javascript-async-and-await)
- [x] @Chinchilla15 [Asynchronous Code](https://www.theodinproject.com/lessons/node-path-javascript-asynchronous-code)
- [x] @MiguelHigueraDev [Working with APIs](https://www.theodinproject.com/lessons/node-path-javascript-working-with-apis)

@JoshDevHub 

### Path

Node / JS

### Lesson Url

javascript/asynchronous_javascript_and_apis

**Comment by VMadhuranga** [comment]
I would like to help out here

**Comment by JoshDevHub** [comment]
I've assigned you the lesson on async/await @VMadhuranga 

Thank you for volunteering :raised_hands:

**Comment by Chinchilla15** [comment]
Hi! I would like to help with any lesson available :)

**Comment by JoshDevHub** [comment]
@Chinchilla15 Thank you for volunteering!

I've assigned you the Asynchronous Code lesson.

**Comment by MiguelHigueraDev** [comment]
Hello I would like to contribute if you still need any help!

**Comment by JoshDevHub** [comment]
Absolutely @MiguelHigueraDev 

I've assigned you the Working with APIs lesson

**Comment by JoshDevHub** [comment]
Thanks everyone for the contributions :muscle:
